Description: in build environment udp multicast is sometimes not permitted
Forwarded: https://github.com/libuv/libuv/pull/1689
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2017-12-29
--- a/test/test-udp-multicast-interface.c
+++ b/test/test-udp-multicast-interface.c
@@ -44,7 +44,7 @@
 
 static void sv_send_cb(uv_udp_send_t* req, int status) {
   ASSERT(req != NULL);
-  ASSERT(status == 0 || status == UV_ENETUNREACH);
+  ASSERT(status == 0 || status == UV_ENETUNREACH || status == UV_EPERM);
   CHECK_HANDLE(req->handle);
 
   sv_send_cb_called++;
--- a/test/test-udp-multicast-ttl.c
+++ b/test/test-udp-multicast-ttl.c
@@ -44,7 +44,7 @@
 
 static void sv_send_cb(uv_udp_send_t* req, int status) {
   ASSERT(req != NULL);
-  ASSERT(status == 0 || status == UV_ENETUNREACH);
+  ASSERT(status == 0 || status == UV_ENETUNREACH || status == UV_EPERM);
   CHECK_HANDLE(req->handle);
 
   sv_send_cb_called++;
